### ISSUE TITLE
Custom loss function error

### ISSUE THREAD

**Issue by Se-Boruk** [Ciapser/NeuroUtils#15]
While specifying own loss function and applying it to the model it compiles correctly, however script responsible for creating model parameters file does not read it correctly as it expects string name of the loss function and not custom function itself. 
That leads to error which stops creating model parameter file and in consequence doesn't allow creating dedicated model folder and training it.
